apps/docs/pages/guides/auth/oauth-with-pkce-flow-for-ssr.mdx

```diff
@@ -0,0 +1,192 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'OAuth with PKCE flow for SSR',
+  description: 'Learn how to configure OAuth authentication in your server-side rendering (SSR) application to work with the PKCE flow.',
+}
+
+### Install Supabase Auth Helpers
+
+The Auth Helpers will assist you in implementing user authentication within your server-side rendering (SSR) framework.
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="nextjs"
+>
+<TabPanel id="nextjs" label="NextJS">
+```bash
+npm install @supabase/auth-helpers-nextjs @supabase/supabase-js
+```
+</TabPanel>
+<TabPanel id="sveltekit" label="SvelteKit">
+```bash
+npm install @supabase/auth-helpers-sveltekit @supabase/supabase-js
+```
+</TabPanel>
+</Tabs>
+
+### Set environment variables
+
+Create an `.env.local` file in your project root directory. You can get your `SITE_URL` and `ANON_KEY` from inside of the [dashboard](https://supabase.com/dashboard/project/_/settings/api).
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="nextjs"
+>
+<TabPanel id="nextjs" label="NextJS">
+```bash .env.local
+NEXT_PUBLIC_SUPABASE_URL=your_supabase_project_url
+NEXT_PUBLIC_SUPABASE_ANON_KEY=your_supabase_anon_key
+```
+</TabPanel>
+<TabPanel id="sveltekit" label="SvelteKit">
+```bash .env.local
+PUBLIC_SUPABASE_URL=your_supabase_project_url
+PUBLIC_SUPABASE_ANON_KEY=your_supabase_anon_key
+```
+</TabPanel>
+</Tabs>
+
+### Setting up the Auth Helpers
+
+When using the Supabase client on the server, you must perform extra steps to ensure the user's auth session remains active. Since the user's session is tracked in a cookie, we need to read this cookie and update it if necessary.
```

Comment:
```suggestion
For SSR, the Supabase client requires extra steps to ensure the user's auth session remains active. Since the user's session is tracked in a cookie, we need to read this cookie and update it if necessary.
```